packages/utils/transformers.ts

```diff
@@ -16,10 +16,11 @@ export function configPersistenceToContext(
 export function configContextToPersistence(
   config: (IConfig | undefined)[],
 ): ConfigStates {
-  const [electionDatabaseId, resultsDatabaseId] = config;
+  const [electionDatabaseId, resultsDatabaseId, notionApiKey] = config;
 
   return {
     electionDatabaseId: electionDatabaseId?.value || '',
     resultsDatabaseId: resultsDatabaseId?.value || '',
+    notionApiKey: notionApiKey?.value || '',
```

Comment:
Maybe could be better to use the `??` operator here ( called null coalescing). Basicaly if the right-hand side operand is null then it executes the left-hand side operand. Essentially it is the same of use `||` but, `??` is a modern way to do that, and has no 'magic' parse boolean parse.